src/ovp_pipeline/commands/_ui_renderers.py

```
@@ -1286,6 +1309,157 @@ def _render_note_page(
     )
 
 
+def _render_lineage_card(
+    lineage: dict | None, *, requested_pack: str = "",
```

Ruff (0.15.12)

[warning] Unused function argument: `requested_pack`

(ARG001)